pkg/controller/control.go

```diff
@@ -642,20 +663,33 @@ func (c *Controller) Start(addresses ...string) error {
 			continue
 		}
 
-		availableBackends[address] = newBackend
-
 		if first {
 			first = false
-			c.size = newSize
 			c.sectorSize = newSectorSize
-		} else if c.size != newSize {
-			availableBackends = map[string]types.Backend{}
-			fatalErr = fmt.Errorf("BUG: Backend sizes do not match %d != %d in the engine initiation phase", c.size, newSize)
-			break
-		} else if c.sectorSize != newSectorSize {
-			availableBackends = map[string]types.Backend{}
-			fatalErr = fmt.Errorf("BUG: Backend sector sizes do not match %d != %d in the engine initiation phase", c.sectorSize, newSectorSize)
-			break
+		}
+
+		if c.sectorSize != newSectorSize {
+			logrus.Warnf("backend %v sector size does not match %d != %d in the engine initiation phase", address, c.sectorSize, newSectorSize)
+			continue
```

Comment:
Do you know why the previous behavior will just error out if any backend is not matched? The new change looks sense to me by just bypass instead.

Reply:
Well, I'm not really sure why the check was not implemented. I guess it might be due to that the case rarely happened and just for the simplicity.
=

Reply:
cc @shuo-wu @joshimoo